ddtrace/llmobs/_integrations/mcp.py

```diff
@@ -17,13 +19,40 @@
 
 log = get_logger(__name__)
 
+MCP_SPAN_TYPE = "_ml_obs.mcp_span_type"
+
 SERVER_TOOL_CALL_OPERATION_NAME = "server_tool_call"
 CLIENT_TOOL_CALL_OPERATION_NAME = "client_tool_call"
 
 
+def _find_client_session_root(span: Optional[Span]) -> Optional[Span]:
```

Comment:
This won't work for distributed traces where the client session is in the upstream service. That's fine right? If so let's add a quick docstring indicating this